Dockerfile

```diff
@@ -182,10 +186,9 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
         libxrender1 \
         libxtst6 \
         netbase \
-        nodejs \
         openjdk-11-jre \
         openssh-client \
-        sbt=$SBT_VERSION \
+        sbt \
```

Comment:
For some our OSS reviews it important that we able to set the SBT to not be the latest. A lot of SBT projects are stuck on 0.13.18 and do not work with sbt 1.x

Reply:
Done